api/lib/network-info-logger.js

```diff
@@ -0,0 +1,27 @@
+const stationIdsSeen = new Set()
+setInterval(() => { stationIdsSeen.clear() }, 1000 * 60 * 60 * 24) // clear every day
```

Comment:
I think this blocks the event loop from exiting when running the tests. 

```
npm t -w api -- -i -g "getRoundStartEpoch|startRoundTracker"
```

Possible solutions that come to my mind:
1. Allow the tests to clear this interval.
2. Initialise `setInterval` in a file not loaded by the tests, e.g. a bin file.

I like the second option more. The problem with the first option is that this file is kind of deep in the tree, and it's not clear which tests are expected to clear the interval.

My proposal:
- In this file, export a function to clear the stations seen
- Setup the interval somewhere else.

This also enables us to test whether the cleaning function works as intended.

Reply:
When you say a bin file, do you mean a new one? Doesn't that need to be run directly as part of a node/npm run command?

Or should we instead include this function call in the bin/spark.js file?

Reply:
Went with the assumption of bin/spark.js, addressed in https://github.com/filecoin-station/spark-api/pull/339/commits/6332977b75dd30bd098b3c751b5efbefb59a2d0e